docs-v2/mint.json

```diff
@@ -93,6 +93,7 @@
                         "integration-templates/notion",
                         "integration-templates/salesforce",
                         "integration-templates/slack",
+                        "integration-templates/wildix-pbx",
```

Comment:
I don't see the corresponding `.mdx` file for this.

Reply:
If you add this line, you must add a dedicated docs page under `docs-v2/integration-templates/wildix-pbx.mdx`. But we're deprecating that folder, so it's best to just remove that line if you don't mind.

Reply:
Removed